### PR TITLE
Added language Keli

### PR DESCRIPTION
**Pull request by wongjiahau**

### CODE REVIEW

**Comment by athas** [comment]
Are you sure that's the right author?

File: index.xhtml [hunk 1]
```diff
@@ -54,6 +54,17 @@
       <p>A high-level object-oriented language using a Rust style borrow checker
         for memory management (no GC).</p>
     </div>
+    
+     1<div class="project">
```
Comment (jamesotron) [inline index.xhtml]: Spurious `1` - typo?

**Comment by jamestmartin** [comment]
That is also not the appropriate use of the author link. The author link should link to your preferred profile page or nothing at all.

**Comment by swordglowsblue** [comment]
Not only has the author name not been updated, the description hasn't been updated either. It still uses Adamant's description, instead of an applicable description to Keli (which is described on its site as "a general purpose programming language where its semantics are mostly based on the pure functional programming model [...] while its syntactic structure is strongly influenced by Smalltalk's message sending syntax...")

**Comment by jamestmartin** [comment]
It's also under the 'A's instead of the 'K's.

**Comment by wongjiahau** [comment]
Sorry for the mistakes I'll create another PR again

**Comment by wongjiahau** [comment]
@jamestmartin Thanks!

**Pull request closed without merging**